### PR TITLE
Initial Commit adding main.cpp and Game_Type.h

### PR DESCRIPTION
**Pull request by subhed**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -0,0 +1,73 @@
+/*  
+	main.cpp by Subhed Chavan, Jayesh Sathe, Gopal Raigoni @ Chico State
+    Github @subhed @Jayesh-sathe @gopalsingh112
+    CSCI-630 
+*/
+
+#include <iostream>
+#include "Game_Type.h"
+#include "human_player.h"
```
Comment (kbuffardi) [inline main.cpp]: We're using Capital_Letter formatting

Comment (subhed) [reply main.cpp]: The file name issue resolved.

Comment (subhed) [reply main.cpp]: The header file name issue resolved.

File: main.cpp [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        return GT;
+        break;
+
+    case 2:
+        GT = Game_Type::dual;
+        return GT;
+        break;
+
+    default:
+        break;
```
Comment (kbuffardi) [inline main.cpp:37]: If invalid input, repeat the prompt/entry

File: main.cpp [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        GM(GT);
+        show_records();
+        break;
+
+    case 2:
+        show_records();
+        break;
+
+    default:
+        break;
```
Comment (kbuffardi) [inline main.cpp:76]: 3 to quit, anything else, print an error message and repeat menu.

Comment (subhed) [reply main.cpp:76]: Feature added to the program.

File: main.cpp (hunk 2 above)
Comment (subhed) [reply main.cpp:37]: Issue fixed for invalid entry.

**Pull request merged**